test/OpenTelemetry.Tests.Stress/Skeleton.cs

```diff
@@ -31,7 +31,7 @@ public partial class Program
     public static void Stress(int concurrency = 0)
     {
 #if DEBUG
-        Console.WriteLine("***WARNING*** The current build is DEBUG which may affect timing!\n");
+        Console.WriteLine($"***WARNING*** The current build is DEBUG which may affect timing!{Environment.NewLine}");
```

Comment:
This is a very weird thing in C#.
Normally folks would put `Console.WriteLine()` (and that's why this overload exists).

Check what .NET runtime repo is doing https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/tests/JIT/Directed/shift/uint64.cs#L48.